src/main/java/mate/academy/model/Book.java

```diff
@@ -16,8 +22,9 @@
 @Setter
 @Entity
 @SQLDelete(sql = "UPDATE books SET is_deleted = true WHERE id=?")
-@SQLRestriction(value = "is_deleted=false")
+@SQLRestriction(value = "is_deleted = false")
 @Table(name = "books")
+@NoArgsConstructor
```

Comment:
as I can see you don't have any constuctor with params, so you can remove this annotation